openstack/cce/v3/clusters/requests.go

```diff
@@ -175,20 +177,35 @@ func GetCert(c *golangsdk.ServiceClient, id string) (r GetCertResult) {
 
 // UpdateOpts contains all the values needed to update a new cluster
 type UpdateOpts struct {
-	Spec UpdateSpec `json:"spec" required:"true"`
+	Metadata *UpdateMetadata `json:"metadata,omitempty"`
+	Spec     UpdateSpec      `json:"spec" required:"true"`
```

Comment:
it's better to switch the order of two parameters

Reply:
sure